internal/transformations/base64decode_test.go

```diff
@@ -31,7 +100,7 @@ func BenchmarkB64Decode(b *testing.B) {
 
 func FuzzB64Decode(f *testing.F) {
 	for _, tc := range b64DecodeTests {
-		f.Add(tc)
+		f.Add(tc.input)
```

Comment:
Interestingly the fuzz test was still set up for this forgiving version, that's nice